app/models/incident.rb

```diff
@@ -34,8 +44,8 @@ def digest_sent_at_consistent
     if digest_sent_at && !digest_worthy
       errors.add(:digest_sent_at, "can't be set if digest_worthy is false.")
     end
-    if digest_sent_at && !resolved_at
-      errors.add(:digest_sent_at, "can't be set if incident is not resolved.")
+    if digest_sent_at && !visible?
```

Comment:
This change resonates with my other remark about "merging" the ifs in the controller permissions check